<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TargetWinInfo</class>
 <widget class="QDialog" name="TargetWinInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>309</width>
    <height>184</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>登录信息</string>
  </property>
  <widget class="QPushButton" name="pushButton_Tyes">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>140</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>281</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QIPLineEdit" name="lineEdit_Tipaddr">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>登录用户名</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>用户名密码</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Tpwd">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>80</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Tusr">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>50</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>IP 地址</string>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QIPLineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">qiplineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
